src/pages/offer/offer.tsx

```diff
@@ -44,101 +43,105 @@ function Offer({cities, offers, reviews, authorizationStatus}: TOfferProps): JSX
 
 
   return (
-    <main className="page__main page__main--offer">
-      <section className="offer">
-        <OfferGallery key={`${currentOffer.id }gallery`} offerId={currentOffer.id} images={currentOffer.images} />
-        <div className="offer__container container">
-          <div className="offer__wrapper">
-            { currentOffer.isPremium && (<div className="offer__mark"><span>Premium</span></div>) }
-            <div className="offer__name-wrapper">
-              <h1 className="offer__name">
-                {currentOffer.title}
-              </h1>
-              <button className="offer__bookmark-button button" type="button">
-                <svg className="offer__bookmark-icon" width="31" height="33">
-                  <use xlinkHref="#icon-bookmark"></use>
-                </svg>
-                <span className="visually-hidden">To bookmarks</span>
-              </button>
-            </div>
-            <div className="offer__rating rating">
-              <div className="offer__stars rating__stars">
-                <span style={{width: '80%'}}></span>
-                <span className="visually-hidden">Rating</span>
+    <Layout page={'offer'}>
```

Comment:
Строки пропсами оформляем так
```suggestion
    <Layout page="offer">
```